### ISSUE TITLE
Windows Client 1.2.1: special sync problem with a single file made every sync failing, and how I solved that finally

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/client#471]
```
tl;dr
user story:
report of a non-existing file in the client sync folder (windows XP)  
which is correct on the server  
but which causes a DoS in the client sync process
how it presumably happened and how it was finally solved
```
- server version 4.7.8 and
- server version 5.0

I run into a sync problem, which I want to describe here, so that it becomes trackable, because I am sure it is a _lurking_ _problem_ of the csync mechanism, which must never happen even under the worst circumstances.

Let me explain, and ask me for clarifications, in case you do not understand the present subject-matter.
#### What did I do ?
- My intention was to make a fresh sync from my server to my windows machine.  
  ( Why ? Because I still have that spooky file.1234567890 problem see https://github.com/owncloud/core/issues/761 <== not forming part of this issue)
- I deleted my "C:\Documente und Einstellungen\Benutzer\ownloud" folder - as far as I remember without having the Sync stopped (which never must cause a problem.  
  Assuming, _arguendo_ that you are right and say that I am not allowed to move folders with a loaded Sync and accidentially just active syncing process in the task bar, then in this case I tell you that the sync mechanism is _per se_ hopelessly broken and unsafe, and would make me filing a different issue to heal that. I do not hope this.
- The Sync started to recreate the folder and subfolders from the server (blue sync indicator in the task bar)
- After a while, I saw a "red x cross" - the sync failed.
- One minute later (I set the sync interval to 1 minute), the sync started and failed again.
- This "sync-download-and-fail" happening for days
- There was no meaningful indication for example the name of a specific file in the information panel in the client.
#### Today I noticed

that the client downloads every minute a (valid) file `foo.zip~<shorthash>` from my server to my client (I see also two .csync-something files). which then is deleted and triggers the sync failure.
- I checked via the web interface, that this zip file is correctly stored on the server, and is for example downloadable without checksum errors
- I am an MD5- and SHA-junkie, so I know what I am doing and saying, if I say, that this file is and was always correct.
#### How I solved that problem
- I went to my web interface (Firefox 19.0.2) and visited my server.
- I downloaded the valid and sane file `foo.zip` through the browser to my windows desktop.
- I dragged and dropped that file in the ownloud folder, where every minute the sync failed (see above)
- Soon later
#### the problem was immediately solved.

Does this say something to you ?

Before you ask, no, I do not have screenshots, or log files, and I deeply apologise for that. honestly.

Perhaps you have enjoyed my reading, then make please a money donation to the owncloud project, or buy me a beer during the next version's release party.
